src/database/schema.js

```diff
@@ -0,0 +1,45 @@
+const graphql = require('graphql');
+
+const {
+  GraphQLObjectType, GraphQLSchema, GraphQLInt, GraphQLString, GraphQLList,
+} = graphql;
+const fetch = require('node-fetch');
+
+const testToken = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJlbWFpbCI6Im1hcmsuc0BvdXJfY29tcGFueS5jb20iLCJwYXNzd29yZCI6Im1hcmt0aGVhZG1pbjQiLCJpYXQiOjE1NDM0MDY0NDZ9.fyJO2wkBnIbYAJwSYaDpq4lVvCUyZMQcvugdvb3Fuiw';
```

Comment:
Keeping tokens in your files submited to git and github is risky practice.
https://wptavern.com/ryan-hellyers-aws-nightmare-leaked-access-keys-result-in-a-6000-bill-overnight

In this case it's not an AWS key, but developing good practices is a important thing from the start.

Consider using https://www.npmjs.com/package/dotenv for storing sensitive data.

Reply:
Ok I will add a card to trello, to learn dotenv